package Day1_ProgrammingElements;

import java.util.*;

public class UniversityDiscountInput {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        double fee, discountPercent;
        System.out.print("Enter fee: ");
        fee = sc.nextDouble();
        System.out.print("Enter discount percent: ");
        discountPercent = sc.nextDouble();
        double discount = fee * discountPercent / 100;
        double finalFee = fee - discount;
        System.out.println("The discount amount is INR " + discount + " and final discounted fee is INR " + finalFee);
    }
}
